client/mysqlbinlog.cc

```diff
@@ -1462,6 +1466,7 @@ Exit_status process_event(PRINT_EVENT_INFO *print_event_info, Log_event *ev,
 end:
   rec_count++;
   last_processed_datetime= ev_when;
```

Comment:
It feels like these two are a tightly coupled pair, to suggest arrange them as a struct (of two members) which
would single-line the pair of the current assignments (not a big deal) but more interestingly this measure would unite "artificially" separated [1](https://github.com/MariaDB/server/blob/review-10.5-Modev-35694/client/mysqlbinlog.cc#L3131)`+`[2](https://github.com/MariaDB/server/blob/review-10.5-Modev-35694/client/mysqlbinlog.cc#L3140) stop-condition checks into something like `if (stop_condition > last_processed)` where the compared two are structs.